localstack/services/transcribe/provider.py

```diff
@@ -342,7 +342,7 @@ def _run_transcription_job(self, args: Tuple[TranscribeStore, str]):
                     {
                         "start_time": unigram["start"],
                         "end_time": unigram["end"],
-                        "type": "pronounciation",
+                        "type": "pronunciation",
```

Comment:
While the others don't have any impact at all (and are only cosmetical / in the code itself), this one actually increases parity with AWS: https://docs.aws.amazon.com/transcribe/latest/APIReference/API_streaming_Item.html
/cc @sannya-singal @ackdav